modules/ROOT/pages/design-mocking-service.adoc

```diff
@@ -1,68 +1,76 @@
+== Before You Begin
 
-----
-#%RAML 1.0
-title: placeholder
-version: 1.0
-baseUri: https://anypoint.mulesoft.com/mocking/api/v1/links/d0422e05-e779-4016-9d9e-027e50cd15a7
-# baseUri: http://jsonplaceholder.typicode.com
-----
+You must have a project containing an API specification in API Designer. Also, you must define at least one endpoint in your API specification and each endpoint must have at least one method.
 
-You can run the mocking service from the text editor, the visual editor, and from Anypoint Exchange. If you want to run the mocking service from Anypoint Exchange after publishing your API specification, see "To Simulate Data in REST APIs" in the Anypoint Exchange documentation. A link to that topic is in the *See also* section at the end of this topic.
+== Simulate a Call to an API using an Internal URL
 
-[NOTE]
+[IMPORTANT]
 ====
-API Designer's public, mocked URI is also visible in Exchange. There, people who discover your API can try it out by using the built-in Exchange mocking service, or by using the public URI either in their code or in a tool for testing API implementations.
+To access the internal URL, you must specify the authorization headers as part of the request URL. To test an API without supplying authorization headers, generate a mocking service public URL for this project.
 ====
 
-== Procedure
-
-* If you are using the text editor in API Designer:
-+
-. Turn on the mocking service by clicking the *Mocking service* slider in the top-right corner of the editor.
-. In the API summary panel, expand completely the resource with the endpoint that you want to test.
+. From API Designer, open the project containing the API you want to test.
 +
-_Result:_ After you expand the resource, you will see one or more buttons. The labels on the buttons match the different request types for the resource.
-. Click the button that matches the request type that you want to simulate.
+You can use either the text editor or the visual editor.
+. Select the root file of your project.
+. Click the *Documentation* icon in the right-hand tool bar.
+. From the drop-down menu, select *Summary*.
+. Click on the name of an end point.
+. If you have defined `baseURL` in your API specification:
 +
-_Result:_ The panel displays information about the request type, its parameters, and the codes and messages that it returns.
-. Click *Try It* in the top-right corner of the editor.
+* Click the *Select server* dropdown and select one of the following:
 +
-_Result:_ The URI for the endpoint is displayed. If the endpoint requires parameters, specify values for them. Also, you can view the headers and add custom headers.
-. Click *Send*.
-+
-_Result:_ The panel displays the result below the *Send* button.
+** baseUri: uses the baseUri as defined in your API specification
```

Comment:
should baseUri have tick marks? `baseUri`?

Reply:
Fixed